File: CoreJava/account-polymorphism/src/account/polymorphism/CurrentAccount.java
package account.polymorphism;

public class CurrentAccount extends Account {

	private static final double OVERDRAFT = -5000.0;

	public CurrentAccount(String n, double b) {
		super(n, b);
		
	}

	@Override
	public void withdraw(double amt) {
		countWithdraw+=1;
		if (balance - amt < OVERDRAFT) {
			System.out.println("Insufficient Balance");
		} else
			balance -= amt;

	}


}

File: C#/OOP/PolymorphismExample/PolymorphismExample/CurrentAccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PolymorphismExample
{
    class CurrentAccount : Account
    {
        private static readonly double OVER_DRAFT = -5000.0;

        public CurrentAccount(String n, double b)
            : base(n, b)
        {
        }


        public override void withdraw(double amt)
        {
                countWithdraw += 1;
                if (balance - amt < OVER_DRAFT)
                {
                    Console.WriteLine("Insufficient Balance");
                }
                else
                    balance -= amt;
            
        }
    }
}
